### DISCUSSION TITLE
Can users add customised labels?

### DISCUSSION THREAD

**Discussion by ohlaris** [cnr-isti-vclab/TagLab#13] [Q&A] [answered]
Hello, 
Thanks for sharing this great tool. Is there a way to add my own set of labels?

Thanks

**Comment by maxcorsini** [comment, marked as answer]
Hello,

For the moment, you have to modify the label name - label color association in the config.json file. 
A color is specified using three numbers representing its RGB components.

In the next releases, we are planning to add a label manager to simplify this operation.

Hope this helps you!
